gpx_test.go

```diff
@@ -611,19 +612,32 @@ func TestTime(t *testing.T) {
 }
 
 func TestParseExamples(t *testing.T) {
-	for _, filename := range []string{
-		"testdata/ashland.gpx",
-		"testdata/fells_loop.gpx",
-		"testdata/mystic_basin_trail.gpx",
-	} {
-		t.Run(filename, func(t *testing.T) {
+	err := godirwalk.Walk("testdata", &godirwalk.Options{
```

Comment:
Please can you use [`io/fs.WalkDir`](https://pkg.go.dev/io/fs#WalkDir) instead of `godirwalk.Walk`. This avoids introducing a dependency for functionality that is already in the standard library.

Reply:
Fix